### PR TITLE
Replace most refs to msprime to refer to tskit

### PR DESCRIPTION
**Pull request by hyanwong**
And add @brianzhang01 's installation problems. Fixes https://github.com/tskit-dev/tsinfer/issues/177

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -36,3 +32,25 @@ first using `venv <https://docs.python.org/3/library/venv.html>`_::
...
     (tsinfer-venv) $ tsinfer --help
+
+.. _sec_installation_installation_problems:
+
++++++++++++++++++++++
+Installation problems
++++++++++++++++++++++
+
+
+There are situations (usually where the GSL libraries are not in the
```
Comment (jeromekelleher) [inline docs/installation.rst]: We shouldn't be installing msprime at all now though, right? So, GSL shouldn't come into it at all.

**Comment by jeromekelleher** [comment]
Looks good, thanks. There should be no reference to msprime in the code any more though, so we shouldn't pull it in during install.

**Comment by hyanwong** [comment]
Ah, I assumed tskit used GSL too. If not, I'll simply remove this bit of "Installation problems". OK?

**Comment by jeromekelleher** [comment]
Merging, thanks @hyanwong.

**Pull request merged**